feat: Trailing Comment Whitespace Lint (#177)
stjude-rust-labs/wdl: wdl-lint/CHANGELOG.md

## Unreleased

### Added

> * Added Trailing Comments lint rule as an extension of the `Whitespace` rule ([177](https://github.com/stjude-rust-labs/wdl/pull/177))

Reviewer: ```suggestion
* Added comments to the trailing whitespace check of the `Whitespace` rule ([177](https://github.com/stjude-rust-labs/wdl/pull/177))
```
just nit picking over language 😄
